packages/shared-ui/src/flowHandler/blocks/PasskeyVerifyBlock.ts

```diff
@@ -70,9 +72,13 @@ export class PasskeyVerifyBlock extends Block<BlockDataPasskeyVerify> {
   }
 
   async passkeyLogin() {
+    this.#passkeyAborted = false;
+
     const res = await this.app.authProcessService.loginWithPasskey();
     if (res.err) {
-      this.updateScreen(ScreenNames.PasskeyError);
+      if (!this.#passkeyAborted) {
```

Comment:
we should add a comment here to explain why we need this check (I think we should add more comments in general for code that does something that is hard to understand)